pykern/pkcli/test.py

```diff
@@ -104,7 +104,12 @@ def _find(paths):
     i = re.compile(r'(?:_work|_data)/')
     res = []
     cwd = pkio.py_path()
-    for t in paths or ('tests',):
+    d = paths
```

Comment:
Change the condition order so that it can accept arguments in the tests directory (there should be a test for this):

```py
if not paths:
     p = pkio.py_path()
     if p.basename != _SUITE_D
         p = _SUITE_D
     paths = (p,)
for t in paths:
```

Reply:
Nice. I was wondering about how to best structure those conditionals